BashCompletion/README.md

```diff
@@ -0,0 +1,46 @@
+# Bash Completion
+
+## Installation
+
+    $ brew install bash-completion
```

Comment:
These $ example lines can be in triple-backtick sections with the `console` keyword.

Reply:
I think it's not a problem using $ as long as it's only one line